networkit/cpp/centrality/DynApproxBetweenness.cpp

```diff
@@ -7,172 +8,316 @@
  */
 
 #include <networkit/centrality/DynApproxBetweenness.hpp>
+#include <networkit/auxiliary/Log.hpp>
+#include <networkit/auxiliary/NumericTools.hpp>
 #include <networkit/auxiliary/Random.hpp>
+#include <networkit/centrality/Centrality.hpp>
+#include <networkit/components/StronglyConnectedComponents.hpp>
 #include <networkit/distance/Diameter.hpp>
-#include <networkit/graph/GraphTools.hpp>
-#include <networkit/distance/DynDijkstra.hpp>
 #include <networkit/distance/DynBFS.hpp>
-#include <networkit/auxiliary/Log.hpp>
-#include <networkit/auxiliary/NumericTools.hpp>
-
+#include <networkit/distance/DynDijkstra.hpp>
+#include <networkit/graph/GraphTools.hpp>
 
+// networkit-format
 namespace NetworKit {
 
-DynApproxBetweenness::DynApproxBetweenness(const Graph& G, const double epsilon, const double delta, const bool storePredecessors, const double universalConstant) : Centrality(G, true),
-storePreds(storePredecessors), epsilon(epsilon), delta(delta), universalConstant(universalConstant) {
+DynApproxBetweenness::DynApproxBetweenness(const Graph &G, double epsilon, double delta,
+                                                   bool storePredecessors, double universalConstant)
+    : Centrality(G, true), epsilon(epsilon), delta(delta), storePreds(storePredecessors),
+      universalConstant(universalConstant) {}
+
+inline count DynApproxBetweenness::DFS(node u, count j, Graph &sccDAG,
+                                           std::vector<bool> &marked, std::vector<node> &sorted) {
+    marked[u] = true;
+    sccDAG.forNeighborsOf(u, [&](node v) {
+        if (!marked[v]) {
+            marked[v] = true;
+            j = DFS(v, j, sccDAG, marked, sorted);
+        }
+    });
+    j--;
+    sorted[j] = u;
+    return j;
 }
 
+count DynApproxBetweenness::computeVDdirected() {
+    clock_t begin = clock();
```

Comment:
I think we should not have profiling code in the master branch, it makes the code harder to read for no good reason. Profiling should be done separately.